koku/api/settings/settings.py

```diff
@@ -257,9 +263,12 @@ def _tag_key_handler(self, settings):
                             updated[ix] = True
 
                     for rm_tag in remove_tags:
+                        LOG.info(f"Updating " + provider_name + " tag '" + rm_tag.key + "': DISABLED")
```

Comment:
I understand wanting to log an exception, but don't know that we want to log each and every change? Did someone request we do that?

Reply:
None of these logs are going to show in the UI console, this is server side logging. It is extremely helpful to have logging information when API endpoints are called. We were already logging all this for AWS I just added logging so we are now tracking for the other providers.

Reply:
Perhaps if this worked off of a debug flag, but otherwise I have to disagree. 

We don't want to potentially fill the log with each and every change. Just moving items from one list to another will generate messages.

Reply:
These are only being logged during a **POST** call to the endpoint, not when just moving items around in the Dual list selector.  You have to hit the Save button and all changes that are made are recorded. This same logging has been happening for AWS I just made sure we are capturing the data for the other providers as well. When calling the end point is it is not a bad idea to log the the changes that are happening, this allows you to verify that state being changed is actually being saved properly in the backend(databases, etc...).

 I can certainly understand if you thought they were being logged during the process of just moving items from list to list in the component. This would just file the logs with erroneous information...

Reply:
This is true if the number of tags is a very large amount it would flood the logs. Will track total s instead of specifics